src/status_im/browser/core.cljs

```diff
@@ -150,21 +155,29 @@
                              :history new-history
                              :history-index new-index)))))
 
+(defmulti storage-gateway :namespace)
+
+(defmethod storage-gateway :ipfs
+  [{:keys [hash]}]
+  (let [base32hash (-> (.encode js-dependencies/hi-base32 (alphabase.base58/decode hash))
+                       (string/replace #"=" "")
+                       (string/lower-case))]
+    (str "https://" base32hash ".infura.status.im")))
+
+(defmethod storage-gateway :swarm
```

Comment:
it seems like you didn't add support for swarm in contenthash ns so is it necessary?

Reply:
Not harmful! I am sure you are excited to add swarm support for TtT :)

Reply:
yes but definitely not like this from the `browser.core` namespace